pydash/pydash_app/dashboard/dashboard.py

```diff
@@ -0,0 +1,39 @@
+
+import uuid
+import persistent
+
+
+class Dashboard(persistent.Persistent):
+    """
+
```

Comment:
Are you planning to add documentation later?

Reply:
I'll do that later. Probably when the rest of the implementation is finished.